ide-support/revliburl.livecodescript

```diff
@@ -73,6 +80,49 @@ on libraryStack
    initialiseCustomProps
 end libraryStack
 ------------------------------------------------
+-- CW-2016-06-11: [[ External driver support ]] Add function to select external driver for use with libUrl.
+## Set driver to use for libUrl
+on libUrlSetDriver pDriver
```

Comment:
I think we will have to special-case it for now - I was thinking the easiest thing would be for it to have a handler like `revExtensionDependencies` which could be dispatched to each included library. This could be done in `revSBUpdateForDependencies` perhaps.